tip-wlan/charts/opensync-gw-cloud/values.yaml

```diff
@@ -77,6 +77,8 @@ service:
   port5: 5005
   name5: debug
   nodePort5: 26
+  annotations: {}
+  nodePort_static: true
```

Comment:
Would be worthwhile to add a comment what nodePort_static means here (why is it defaulted to true) and the intention for it's use.